jwst/extract_1d/ifu.py

```diff
@@ -81,9 +81,11 @@ def ifu_extract1d(input_model, refname, source_type):
         flux = net.copy()
         net[:] = 0.
         log.info("Data have been flux calibrated; setting net to 0.")
+        data_units = 'mJy'
     else:
         flux = np.zeros_like(net)
         log.info("Data have NOT been flux calibrated; setting flux to 0.")
+        data_units = 'mJy'
```

Comment:
Shouldn't the units be 'DN/s' for this case (not flux calibrated)?

Reply:
Of course. Copy-n-paste error. Good eye!